pkg/config/config.go

```diff
@@ -1126,3 +1134,20 @@ func (c *Config) ActiveDestination() (uri, identity string, err error) {
 	}
 	return "", "", errors.New("no service destination configured")
 }
+
+// ImageCopyTmpDir default directory to store tempory image files during copy
+func (c *Config) ImageCopyTmpDir() (string, error) {
+	path := func() string {
+		if path, found := os.LookupEnv("TMPDIR"); found {
+			return path
+		}
+		if c.Engine.ImageCopyTmpDir == "storage" {
```

Comment:
Something-like
```suggestion
		switch {
		// Absolute paths are accepted as is
		case filepath.IsAbs(c.Engine.ImageCopyTmpDir): return c.Engine.ImageCopyTmpDir
		// Other strings are treated as special keywords
		case c.Engine.ImageCopyTmpDir == "storage": return …
		default: return "", errors.Errorf("unknown image_copy_dir value %q (relative paths are not accepted)")
		}
```
is equivalent in purpose, but explicit about what this is doing.

(The final check, which also verifies that TMPDIR/graphRoot is absolute, is an interesting sanity check, but doesn’t express the _intent_ to split the namespace.)